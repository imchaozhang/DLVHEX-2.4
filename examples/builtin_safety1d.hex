#maxint=10.

boo(X) :- #int(X).
