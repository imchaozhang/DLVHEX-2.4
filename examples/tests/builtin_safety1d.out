{boo(0), boo(1), boo(2), boo(3), boo(4), boo(5), boo(6), boo(7), boo(8), boo(9), boo(10)}
